### PR TITLE
Navbar code improvement

### LINKED ISSUE #246: Use Cloudinary Images, Code & UX Improvements
- Currently, we are using the website-static repo for getting the user images We need to use images from Cloudinary as the website-static repo will not be used anymore in the coming future.
- Redirecting the user without letting them know anything? The user comes to the crypto site, doesn't have a cookie, gets redirected to a GitHub login page?! Wouldn't the user be confused? The redirection should be in a new tab, right? After logging in with GitHub, we redirect the user to realdevsquad.com. That's a dead end. The Crypto page is lost now.
- Repetition of code is there, we can use an array of objects here.
image

If any doubts please ask here.

### PR DESCRIPTION
**Pull request by shubham-y**
Closes #246 

Add on: Update ecmaVersion in ESLint config(No error found on lint check)

### CODE REVIEW

File: components/NavBar/index.js [hunk 1]
```diff
@@ -33,14 +33,15 @@ const NavBar = () => {
         })
         .then((responseJson) => {
           if (responseJson.incompleteUserDetails) {
-            return window.location.replace(
-              'https://my.realdevsquad.com/signup'
-            );
+            window.open('https://my.realdevsquad.com/signup', '_blank');
```
Comment (swarajpure) [inline components/NavBar/index.js:36]: Should we show a prompt to inform the user what's happening and then let them take the call? "Looks like you haven't filled in your profile information, would you like to do that?"

Comment (shubham-y) [reply components/NavBar/index.js:36]: We can show a prompt but then we will not be able to persuade the users to fill in their profile information

File: components/NavBar/index.js [hunk 2]
```diff
@@ -33,20 +32,20 @@ const NavBar = () => {
...
           }
           setIsLoggedIn(true);
           setUserData({
             userName: responseJson.username,
             firstName: responseJson.first_name,
+            profilePicture: responseJson.picture
+              ? responseJson.picture.url
```
Comment (rohan09-raj) [inline components/NavBar/index.js]: I think we can use optional chaining here. What do you think ??

File: constants.js [hunk 3]
```diff
@@ -9,13 +9,41 @@ const CURRENCIES = {
...
+  {
+    name: 'Crypto',
+    url: 'https://crypto.realdevsquad.com/',
+    isActive: true,
+  },
+  {
+    name: 'Status',
+    url: 'https://status.realdevsquad.com/',
+    isActive: false,
+  },
```
Comment (rohan09-raj) [inline constants.js:44]: Great job in here 👏🏻

File: components/NavBar/index.js (hunk 2 above)
Comment (shubham-y) [reply components/NavBar/index.js]: Yes, combining Optional chaining and Nullish coalescing operator will work
```js
responseJson.picture?.url ?? DEFAULT_AVATAR
```
I will make the changes

Comment (rohan09-raj) [reply components/NavBar/index.js]: Yes, that would be great.

Comment (shubham-y) [reply components/NavBar/index.js]: Done

**Review by akshay1502 — APPROVED** [review]

**Review by rohan09-raj — APPROVED** [review]
Looks good to me 🥳 Great job @shubham-y

**Pull request merged**